objc/runtime.h

```diff
@@ -26,6 +26,12 @@ extern "C" {
 #	endif
 #endif
 
+#if __has_attribute(ns_returns_retained)
```

Comment:
I think the following would work to resolve the warnings when importing this from non-ObjC files:
```suggestion
#if defined(__OBJC__) && __has_attribute(ns_returns_retained)
```